ui/src/components/FacetCard.css

```diff
@@ -19,7 +19,7 @@
 
 .subHeader {
     margin-top: 5px;
-    color: gray;
+    color: #553A70;
```

Comment:
I prefer gray over purple. If it's purple, it looks like the most important part of the card.

Reply:
Done